dpath/common.py

```diff
@@ -0,0 +1,28 @@
+import sys
+if sys.version_info < (3,):
+    def u(x):
```

Comment:
Can we name this something more descriptive? Leaving it as u(), plus using it later without the fully qualified path (e.g. "u(somestr)" as opposed to "dpath.common.u(somestr)") offers confusion re: the builtin unicode u() shorthand.